code/game/objects/items/weapons/melee/melee_misc.dm

```diff
@@ -79,6 +79,129 @@
+	The pinnacle of Moghes' combat technology, it combines all of this knowledge into a massive, heavy slab of alloyed metal that most \
+	species find difficult to lift, let alone use in any sort of fight."
+	. += ""
+	. += "Actually a little lightweight for its size, a Breach Cleaver is unmatched in combat utility as a weapon, a tool for getting into\
+	places and as a slab of armour for the wielder. The leather of the Kar'oche beast, a predator native to Moghes, binds the hilt, \
+	allowing it to be gripped securely by its warrior. The wide blade is often etched with scenes depicting military victories or great hunts."
+
+/obj/item/melee/breach_cleaver/update_icon_state()
+	icon_state = "[base_icon_state]0"
+
+/obj/item/melee/breach_cleaver/proc/wield(obj/item/source, mob/living/carbon/human/user)
+	to_chat(user, "<span class='notice'>You heave [src] up in both hands.</span>")
+	user.apply_status_effect(STATUS_EFFECT_BREACH_AND_CLEAVE)
+	update_icon_state()
+
+/obj/item/melee/breach_cleaver/proc/unwield(obj/item/source, mob/living/carbon/human/user)
+	user.remove_status_effect(STATUS_EFFECT_BREACH_AND_CLEAVE)
+	update_icon_state()
+
+/obj/item/melee/breach_cleaver/attack_obj(obj/O, mob/living/user, params)
+	if(!HAS_TRAIT(src, TRAIT_WIELDED)) // Only works good when wielded
+		return ..()
+	if(!ismachinery(O) && !isstructure(O)) // This sword hates doors
+		return ..()
+	if(SEND_SIGNAL(src, COMSIG_ITEM_ATTACK_OBJ, O, user) & COMPONENT_NO_ATTACK_OBJ)
+		return
+	if(flags & (NOBLUDGEON))
+		return
+	var/mob/living/carbon/human/H = user
+	H.changeNext_move(CLICK_CD_MELEE)
+	H.do_attack_animation(O)
+	H.visible_message("<span class='danger'>[H] has hit [O] with [src]!</span>", "<span class='danger'>You hit [O] with [src]!</span>")
+	var/damage = force_wield
+	damage += H.physiology.melee_bonus
+	O.take_damage(damage * 3, BRUTE, MELEE, TRUE, get_dir(src, H), 30) // Multiplied to do big damage to doors, closets, windows, and machines, but normal damage to mobs.
+
+
+/obj/item/melee/breach_cleaver/attack(mob/target, mob/living/user)
+	armour_penetration_flat = 0
+	if(!HAS_TRAIT(src, TRAIT_WIELDED) || !ishuman(target))
+		return ..()
+
+	var/mob/living/carbon/human/H = target
+	var/obj/item/organ/external/targetlimb = H.get_organ(ran_zone(user.zone_selected))
+
+	switch(user.a_intent)
+		if(INTENT_HELP) // Stamina damage
+			H.visible_message("<span class='danger'>[user] slams [H] with the flat of the blade!</span>", \
+							"<span class='userdanger'>[user] slams you with the flat of the blade!</span>", \
+							"<span class='italics'>You hear a thud.</span>")
+			user.do_attack_animation(H, ATTACK_EFFECT_DISARM)
+			playsound(loc, 'sound/weapons/swordhit.ogg', 50, TRUE, -1)
+			H.AdjustConfused(4 SECONDS, 0, 4 SECONDS)
+			H.apply_damage(40, STAMINA, targetlimb, H.run_armor_check(targetlimb, MELEE))
+			add_attack_logs(user, H, "Slammed by a breach cleaver.", ATKLOG_ALL)
```

Comment:
I would update these attack logs. They should probably log what intent the attack came from, as things like "smashed", "slammed", "cleaved overhead" won't really convey the information properly to administrators. You could also include maybe parentheses as to what they do e.g. (knock down), (knock back), etc.